base/logging.go

```diff
@@ -549,16 +545,24 @@ func LogSyncGatewayVersion() {
 	}
 }
 
+// Consolef logs the given formatted string and args to console and stderr.
+func Consolef(level LogLevel, key LogKey, format string, args ...interface{}) {
+	format = addPrefixes(format, context.Background(), level, key)
+
+	// Log to the console if enabled, and set to something other than stderr.
+	if consoleLogger.Enabled && !consoleLogger.isStderr {
+		consoleLogger.logger.Printf(format, args...)
+	}
+
+	// Log to stderr
+	fmt.Printf(format+"\n", args...)
+}
```

Comment:
As discussed, want to preserve this logging going to Info.  Ideally also want to avoid duplicating this to console when Info is going to console.